README.md

```diff
@@ -9,9 +9,30 @@ permissions that are set on the workspace level, Secret scopes, Notebooks, Direc
 
 See [contributing instructions](CONTRIBUTING.md) to help improve this project.
 
+## Introduction
```

Comment:
Huh, why this is on this PR?

Reply:
Yeah, this slipped in somehow.

Reply:
Wait, this is Doug's commit... which is in a PR still? I am unclear how this is here and how to remove it.

Reply:
Removed.